index.src.html

```diff
@@ -352,32 +344,18 @@ <h3 id="header">
   [[!RFC5234]]:
 
   <pre class="abnf" link-type="grammar" dfn-type="grammar">
-    Report-To = <a>json-field-value</a>
-                ; See Section 2 of [[HTTP-JFV]], and Section 2 of [[RFC7159]]
+    Clear-Site-Data = "Clear-Site-Data" ":" <a>clear-site-data-value</a>
+    <dfn>clear-site-data-value</dfn> = OWS 1#(<a>type</a> OWS)
+    <dfn>type</dfn> = atom
```

Comment:
Updated, but with some changes:

-> From what I have read in other specs so far, the definition of a header always includes the key and separator (i.e. "Clear-Site-Data" ":"), not just the value
-> That's why I have the token "clear-site-data-value", which stands for the header value only
-> The "data-type" and "extension-type" tokens do not need the suffix "-value" (they represent the datatypes, not the header value as a whole, so the name is expressive enough)
-> I still think we need to specify OWS; I don't think spaces around tokens should be a syntax error
-> You suggested that alpha is wrapped in anchors; do you want it to link to RFC5234?